.github/CODEOWNERS

```diff
@@ -1,3 +1,11 @@
 # https://help.github.com/en/articles/about-code-owners
 
 /custom/aaa/ @emord
+
+/static/ @orangejenny
+/templates/ @orangejenny
```

Comment:
I'm not sure how to do what I think you're trying to do correctly, but I think this will only notify you if someone made a `static` or `templates` folder at the root of the repo

Reply:
oops - switched to file types, that'll achieve the same goal